### DISCUSSION TITLE
Üniversitede okuyan öğrenciler ve dersleri sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#26] [Zeka Sorusu]
Üniversitede okuyan, Zeynep, Ali ve Burcu adlı 3 öğrenci toplam 5 farklı derse kayıtlıdır; Kimya, Fizik, Matematik, Programlama ve Türkçe. Her öğrenci en az iki derse kayıtlıdır. Üçten fazla derse kayıtlı öğrenci yoktur. Derslerle ilgili, ek olarak, aşağıdakiler bilinmektedir.

● Öğrencilerden sadece biri matematik almaktadır ve bu öğrenci Kimya almamaktadır.
● Ali’nin aldığı bütün dersleri Zeynep de almaktadır.
● Türkçe alan her öğrenci Fizik de almaktadır.

Soru 1 : Eğer Burcunun Kimya aldığını biliyorsak, Ali hangi dersi almak zorundadır?

Soru 2 : Alinin 3 ders aldığı biliniyorsa, Burcu hangi dersi almak zorundadır?

**Comment by Eceg25** [comment]
1. Burcu kimya alıyorsa o zaman Zeynep matematik almak zorunda kalır. Buna göre Ali matematik ve kimya alamayacağı ve sadece programlama alırsa tek bir ders olacağı için Türkçe' yi almak zorundadır.
2. Matematik almak zorundadır.
